### ISSUE TITLE
Missing reference interpolation breaks build

### ISSUE THREAD

**Issue by simoncoggins** [anvilco/spectaql#337]
I realise this is probably intentional at the moment but if the input schema contains a comment of the form ```{{text}}``` where text is not a valid reference the the build fails with an error:

```
error: Task error: [Error: Unsupported interpolation encountered: str
  at /var/www/totara/src/main/node_modules/spectaql/dist/themes/default/helpers/interpolateReferences.js:17:13
  at String.replace (<anonymous>:null:null)
  at Object.interpolateReferences (/var/www/totara/src/main/node_modules/spectaql/dist/themes/default/helpers/interpolateReferences.js:13:16)
  at Object.wrapper (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
  at eval (<anonymous>:11:344)
  at prog (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
  at execIteration (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)
  at Object.<anonymous> (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:61:13)
  at Object.wrapper (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
  at eval (<anonymous>:9:49)
  at Object.prog [as fn] (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
  at Object.<anonymous> (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/helpers/if.js:29:22)
  at Object.wrapper (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
  at Object.eval (<anonymous>:8:50)
  at main (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/runtime.js:208:32)
  at ret (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/runtime.js:212:12)
  at ret (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/compiler/compiler.js:519:21)
  at Object.invokePartial (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/runtime.js:334:12)
  at Object.invokePartialWrapper [as invokePartial] (/var/www/totara/src/main/node_modules/handlebars/dist/cjs/handlebars/runtime.js:84:39)
  at Object.eval (<anonymous>:26:28)
[snip]
```

We have a situation where someone used that syntax in a comment to refer to something else entirely:

```
     """
     ids argument is in the form "$identifier, $component" the same as in {{str}} mustache helper
     """
```

we can change this reference of course but it opens us the possibility that developers might accidently add a comment which breaks the docs build without realising it.

I was wondering if it's necessary to break the build or whether instead it could just throw a warning about unsupported interpolations but just ignore them and continue the build?

**Comment by newhouse** [comment]
Option to change this is in `main` and will be published soon on NPM.

**Comment by newhouse** [comment]
In `main` and published in `v1.2.2`.

**Comment by simoncoggins** [comment]
Many thanks, that looks great.
